src/directLine.ts

```diff
@@ -214,4 +226,92 @@ export class DirectLine implements IBotConnection {
             .delay(5 * 1000)
         );
     }
+
+    /**
+     * Gets an observable of streamURL by requestings a new one from the conversation APIs. 
+     * @return An Observable of stream url.
+     */
+    private getWebSocketStreamURL$(): Observable<string> {
+        return this.connectionStatus$
```

Comment:
let's call this webSocketURL$ ('$' is a shorthand for Stream, so it's redundant to use both)